CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## [12.0.0](https://github.com/theforeman/puppet-candlepin/tree/12.0.0) (2021-04-26)
+
+[Full Changelog](https://github.com/theforeman/puppet-candlepin/compare/11.2.0...12.0.0)
+
+**Breaking changes:**
+
+- Add static defaults for parameters [\#182](https://github.com/theforeman/puppet-candlepin/pull/182) ([ehelms](https://github.com/ehelms))
```

Comment:
I was not clear why this change got labeled backwards incompatible.

Reply:
I wasn't positive, but since candlepin::params is not private, doesn't that actually change the API?

An user who previously did something like

```
class { 'candlepin::params':
  some_param => $some_value,
}

include candlepin
```

would find that this no longer works

Reply:
Tis private -- https://github.com/theforeman/puppet-candlepin/blob/master/manifests/params.pp#L3

Reply:
Ah, it is in the comments.

OK, I will re-label and make the update. Thanks!